c7n/resources/efs.py

```diff
@@ -241,3 +270,129 @@ def fetch_resources_lfc(self, resources):
             except client.exceptions.FileSystemNotFound:
                 continue
         return resources
+
+
+@ElasticFileSystem.filter_registry.register('has-root-access')
+class HasRootAccess(Filter):
```

Comment:
so all three of these are variants on underlying use case of matching a statement in a policy, rather than construct a filter for every variation of a policy statement, a has-statement filter might be more appropriate.

Reply:
@kapilt, does that mean that https://github.com/cloud-custodian/cloud-custodian/pull/5632 is sounding better :)

Reply:
That one indeed looks good. @kapilt @JohnHillegass  is there any major blocks for going further with it? Cause if so, I'd be able to just add that filter in the EFS resource.